assets/react/search.jsx

```diff
@@ -15,6 +15,8 @@ ReactDOM.render(
     initialState={initialState}
     searchSubject={'learningCircles'}
     Browse={BrowseLearningCircles}
+    origin={'http://localhost:8000'}
```

Comment:
@s-kennedy I think this is the cause for the tests failing. The test suite doesn't make requests to the live server, it mocks the requests, but it mocks based on URL

Reply:
@dirkcuys good catch, I removed this line but the tests are still failing.